### PR TITLE
Allow multi Java Options + fix all license headers

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

File: generator/api/src/test/java/io/fabric8/maven/generator/api/BaseGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package io.fabric8.maven.generator.api;
+/*
+ *
+ * Copyright 2016 Roland Huss
```
Comment (davsclaus) [inline generator/api/src/test/java/io/fabric8/maven/generator/api/BaseGeneratorTest.java]: We should use the RH license header file. Also its more common to have the lic header before package

Comment (rhuss) [inline generator/api/src/test/java/io/fabric8/maven/generator/api/BaseGeneratorTest.java]: ah, excellent point :)

Will update my IDE prefs accordingly.

**Comment by rhuss** [comment]
[merge]

**Comment by rhuss** [comment]
[merge]

**Pull request merged**